models/enterprise_sonic/prefix_lists/sonic_prefix_lists.yml

```diff
@@ -0,0 +1,82 @@
+---
+GENERATOR_VERSION: '1.0'
+
+ANSIBLE_METADATA: |
+    {
+      'metadata_version': '1.1',
+      'status': ['preview'],
+      'supported_by': 'community',
+      'license': 'Apache 2.0'
+    }
+NETWORK_OS: sonic
+RESOURCE: prefix_list
+COPYRIGHT: Copyright 2019 Red Hat
+
+DOCUMENTATION: |
+  module: sonic_prefix_lists
+  version_added: "1.2.0"
+  author: "Kerry Meyer (@kerry-meyer)"
+  short_description: prefix list configuration handling for SONiC
+  description:
+    - This module provides configuration management for prefix list parameters on devices running SONiC.
+  options:
+    config:
+      description:
+        - Specifies a list of prefix set configuration dictionaries
+      type: list
+      elements: dict
+      suboptions:
+        name:
+          description:
+            - name of a prefix set (a list of prefix entries)
```

Comment:
Maybe you'd want to capitalize this line for consistency.

Reply:
Thanks for the reminder. I have capitalized that line.